src/main/io/gps.h

```diff
@@ -91,6 +96,7 @@ typedef struct gpsConfig_s {
     gpsAutoConfig_e autoConfig;
     gpsAutoBaud_e autoBaud;
     gpsDynModel_e dynModel;
+    gpsUbloxUseGalileo_e ubloxUseGalileo;
```

Comment:
I'd just use a boolean here. Having an enum for just `0` and `1` complicates the code and it seems unlikely that we'll want to add more states to this field in the future.